common/js/src/messaging/message-channels-pool.js

```diff
@@ -0,0 +1,78 @@
+/** @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview This file containts the pool that keeps track of all the
+ * Closure-style AbstractChannel's used for messaging
+ * (PortMessageChannel & SingleMesageBasedChannel).
+ */
+
+goog.provide('GoogleSmartCard.MessageChannelsPool');
+
+goog.require('GoogleSmartCard.Logging');
+goog.require('goog.log.Logger');
+goog.require('goog.messaging.AbstractChannel');
+goog.require('goog.structs.Map');
+
+goog.scope(function() {
+
+/** @const */
+var GSC = GoogleSmartCard;
+
+/**
+ * This class is a pool which keeps track of Closure-style AbstractChannel's.
+ * @constructor
+ */
+GSC.MessageChannelsPool = function() {
```

Comment:
I'm not that good at English grammar, but searching through Chromium repo show that singular form should be used here.